app/web/src/components/AssetDetailIntrinsicInput.vue

```diff
@@ -30,17 +28,38 @@
           :checked="selectedFilter === 'unset'"
           @select="selectFilter('unset')"
         />
+        <span class="pl-xs text-neutral-500">Identity</span>
         <DropdownMenuItem
           label="Bind to Input Socket"
           checkable
-          :checked="selectedFilter === 'inputSocket'"
-          @select="selectFilter('inputSocket')"
+          :checked="selectedFilter === 'inputSocketForIdentity'"
+          @select="selectFilter('inputSocketForIdentity')"
         />
         <DropdownMenuItem
           label="Bind to Prop"
           checkable
-          :checked="selectedFilter === 'prop'"
-          @select="selectFilter('prop')"
+          :checked="selectedFilter === 'propForIdentity'"
+          @select="selectFilter('propForIdentity')"
+        />
+        <span v-if="normalizeToArrayFuncId" class="flex pl-xs text-neutral-500"
+          >Normalize to Array</span
+        >
+        <span v-else class="flex pl-xs text-neutral-500"
+          >Normalize to Array (Regenerate to Install)</span
+        >
+        <DropdownMenuItem
+          label="Bind to Input Socket"
+          checkable
+          :disabled="!normalizeToArrayFuncId"
+          :checked="selectedFilter === 'inputSocketForNormalizeToArray'"
+          @select="selectFilter('inputSocketForNormalizeToArray')"
+        />
+        <DropdownMenuItem
+          label="Bind to Prop"
+          checkable
+          :disabled="!normalizeToArrayFuncId"
+          :checked="selectedFilter === 'propForNormalizeToArray'"
+          @select="selectFilter('propForNormalizeToArray')"
```

Comment:
Inform the user to regenerate and gray-out/disable the selectable options if the new intrinsic func is not yet available. This is true for old workspaces without any new imports.